src/reference/01-General-Info/04-Bintray-For-Plugins.md

```diff
@@ -115,6 +111,28 @@ API key for future use.
 *NOTE: We have to do this before we can link our package to the sbt
 org.*
 
+### User Documentation
+
+You should add a few lines to your plugin's README file so users know how to modify their `project/plugins.sbt` in order to use your plugin.
+SBT plugins always use Ivy style patterns, so the resolver incantation is a bit verbose.
+As always, the first component in the resolver is just a human-friendly name for the resolver.
```

Comment:
After reading, I think that the topic of BinTray linking is out of scope for this page. This discussion belongs wherever resolvers are discussed. If that section already has a discussion of working with BinTray and linking I'll point to it and add whatever comments seem appropriate.